masschallenge/semantic-ui-theme
# [AC-7932]: P2: Fix misbehaving CSS due to !important flag

Changes introduced in [AC-7932](https://masschallenge.atlassian.net/browse/AC-7932):
- Removed custom styles

**How to test:**

- On this branch and the [sibling PR](https://github.com/masschallenge/accelerate/pull/2759).
- Login as a mentor and visit https://test5.masschallenge.org/newofficehours and create or edit an existing office hour
- Set the end time to be equal to or earlier than the start time so that the UI displays an error
- Notice that the error dropdown style is consistent with the [standard error handling provided by semantic-UI.](https://react.semantic-ui.com/modules/dropdown/#states-error)
- Also notice that the custom CSS has been removed.
